.changeset/chatty-clocks-kick.md

```diff
@@ -0,0 +1,5 @@
+---
+"@directus/app": patch
+---
+
+Added overflow scrollbars in manual flow's confirmation dialog
```

Comment:
```suggestion
Added overflow scrollbars in v-card component
```

As we moved the fix to v-card, I believe this change should describe the current PR clearer 👍 (of course we can update this afterwards if there's a better way to describe it 😄)